3.process_cfret_features/README.md

```diff
@@ -1,12 +1,16 @@
 # 3. Processing Extracted Single Cell and Image Features from CellProfiler
 
 In this module, we present our pipeline for processing outputted `.sqlite` files with single cell and image features from CellProfiler.
-The processed features are saved into compressed `.csv.gz` files for use during statistical analysis for each plate (except for image features).
-We are only extracting image features from the second plate (e.g., Factin_Adjusted plate) as we are planning on continuing with this Cell Painting protocol for further plates.
+The processed features are saved into `parquet` files for use during analysis for each plate.
```

Comment:
```suggestion
The processed features are saved into `parquet` files, which are used for analysis for each plate.
```
I think this is what you mean? Feel free to reject if it is incorrect.

Reply:
Yes that is what I meant and sounds clearer, thanks!